frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -162,10 +179,15 @@ public BranchOrCommitCellRendererComponent(
       IGitMacheteForkPointCommit forkPoint = containingBranch.getForkPoint().getOrNull();
 
       if (commitItem.getCommit().equals(forkPoint)) {
-        append(" ${HEAVY_WIDE_HEADED_RIGHTWARDS_ARROW} fork point ? ", new SimpleTextAttributes(STYLE_PLAIN, Colors.RED));
-        append("commit ", REGULAR_ATTRIBUTES);
+        append(
+            " ${HEAVY_WIDE_HEADED_RIGHTWARDS_ARROW} "
+                + getString("string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.fork-point") + SPACE,
```

Comment:
Hmm why space extracted to a constant? b/c of checkstyle?

Reply:
If so, I'd add a space to `allowedDuplicates` in `MultipleStringLiterals` in config/checkstyle/tree_walker_module_directives.xml

Reply:
Yes, It complains about repeating string